dataflow/src/main/java/org/checkerframework/dataflow/cfg/ControlFlowGraph.java

```diff
@@ -235,7 +237,7 @@ public UnderlyingAST getUnderlyingAST() {
     }
 
     /**
-     * Returns the copied tree-lookup map.
+     * Returns the copied tree-lookup map. Ignores convertedTreeLookup.
```

Comment:
I don't understand the new comment. The method returns a copy of that one field, like `getUnaryAssignNodeLookup` returns a copy of that field.
Is this more a to-do comment that we need a method to also access the `convertedTreeLookup`?

Reply:
The way that the tree-lookup map is accessed is via `getNodesCorrespondingToTree`.  That method makes use of both `convertedTreeLookup` and `treeLookup`.  I'm not sure why there is a need for a public method that makes a copy of `treeLookup`, but any client will be inconsistent with `getNodesCorrespondingToTree`.  So it is useful to provide a warning to clients.